datafusion/sqllogictest/test_files/project_complex_sub_query.slt

```diff
@@ -0,0 +1,72 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+##########
+## Prepare Statement Tests
+##########
+
+statement ok
+CREATE EXTERNAL TABLE t (
+c1 INT NOT NULL,
+c2 INT NOT NULL,
+c3 INT NOT NULL,
+c4 INT NOT NULL,
+c5 INT NOT NULL
+)
+STORED AS CSV
+WITH HEADER ROW
+LOCATION '../core/tests/data/demo.csv';
```

Comment:
since `demo.csv` file is not added to this PR. Following tests fails to produce `PhysicalPlan`. I think, it is better to add this file also. We can still test `LogicalPlan`s in the test by using the flag `set datafusion.explain.logical_plan_only = true;`

Reply:
OK, I'll add that